### ISSUE TITLE
[BUG] near "(": syntax error

### ISSUE THREAD

**Issue by Tencross-Sadachi** [bitdruid/python-wayback-machine-downloader#25]
<ins><b>AmazonLinux2 Python3.8.16</b></ins>

**Describe**
I got 'near "(": syntax error' in SnapshotCollection.py. Any suggestion?

**Snapshot to reproduce**
<a href="https://seesaawiki.jp/overlord">https://seesaawiki.jp/overlord</a>

**Command to reproduce**
waybackup -u https://seesaawiki.jp/overlord -c

**Terminal output**
```
<<< python-wayback-machine-downloader v2.0.2 >>>

No CDX file found to inject - querying snapshots...

Querying snapshots...
-----> seesaawiki.jp/overlord/*
download cdx   : 1.49MB [00:01, 824kB/s]

Inserting CDX data into database...
insert cdx     : 100%|██████████████████████████████████████████████████| 9575/9575 [00:03<00:00, 2974.91 l

Indexing snapshots...

Filtering current snapshots...
-------------------------
!-- Exception:
!-- File: ../../usr/local/lib/python3.8/site-packages/pywaybackup/SnapshotCollection.py
!-- Function: filter_current
!-- Line: 180
!-- Segment: cls.db.cursor.execute(
!-- Description: near "(": syntax error
-------------------------```

**Comment by Tencross-Sadachi** [comment]
No problem with --full option.
But I'd like to get only the latest.

regards,

**Comment by bitdruid** [comment]
sure i will check that out as soon as possible. im a bit stressed because of my thesis but maybe on the weekend

**Comment by bitdruid** [comment]
so i cant reproduce. but could it be that `ROW_NUMBER()` is not included in the sqlite3 version shipped with python 3.6 ? i'm on 3.12.6...  fast checkup online gave me the hint that required sqlite 3.25 should be part of at least python 3.8

on which version are you?
could you check
`python --version`

and also a

`python`
`import sqlite3`
`print(sqlite3.sqlite_version)`

**Comment by Tencross-Sadachi** [comment]
I updated the version of sqlite3 from 3.7 to 3.47(latest), then 'syntax error' is resolved!
Thank you for your responce.
